cinema/views.py

```diff
@@ -118,7 +150,12 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet
+):
```

Comment:
In the `OrderViewSet`, the `queryset` is defined at the class level but you also have a `get_queryset` method that overrides it. Defining a queryset at the class level is redundant when `get_queryset` is used. You should remove the class level `queryset` to avoid confusion. This is a code quality issue.